build-support/bin/_release_helper.py

```diff
@@ -858,15 +859,18 @@ def get_notes() -> str:
                 f"release_{version}",
                 "--notes-file",
                 notes_file,
+                "--title",
+                version,
```

Comment:
Wait, why this? And if this, shouldn't it be `f"release_{version}",`?

Reply:
All of our releases have the title same as the tag. (Actually I don't like this style, but it's what we do...)
E.g. https://github.com/pantsbuild/pants/releases?page=7

Reply:
https://github.com/pantsbuild/pants/pull/20009